Resources/doc/index.rst

```diff
@@ -154,6 +154,8 @@ files thanks to the **object references**. Use the ``addReference()`` method to
 give a name to any object and then, use the ``getReference()`` method to get the
 exact same object via its name::
 
+.. code-block:: php
```

Comment:
Is it syntactically valid to have `::`, then this? I don't like what I see when I click "rich diff"

Reply:
This is correct in terms of grammatical structure.
See this link:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block

Reply:
I'm not talking about the `::` on line 157, but on the one on line 155.

Reply:
"A paragraph consisting of two colons ("::") signifies that the following text block(s) comprise a literal block. The literal block must either be indented or quoted (see below). No markup processing is done within a literal block. It is left as-is, and is typically rendered in a monospaced typeface"

But here we do not need it because the two blocks are defined.
The code was edited.

Reply:
And now the rich diff looks good. Thanks.

Reply:
@greg0ire NP ;)